language/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -7,7 +7,7 @@ use crate::tasks::{
     taskify, InitCommand, PublishCommand, RawAddress, RunCommand, SyntaxChoice, TaskCommand,
     TaskInput, ViewCommand,
 };
-use anyhow::*;
```

Comment:
`anyhow::Ok` was causing me troubles so I had to change it.
```
error[E0532]: expected tuple struct or tuple variant, found function `Ok`
   --> testing-infra/transactional-test-runner/src/framework.rs:419:9
    |
419 |         Ok((mut units, warnings)) => {
    |         ^^ not a tuple struct or tuple variant
    |
help: consider importing one of these items instead
    |
6   | use core::result::Result::Ok;
    |
6   | use std::result::Result::Ok;
    |

error[E0532]: expected tuple struct or tuple variant, found function `Ok`
   --> testing-infra/transactional-test-runner/src/framework.rs:525:9
    |
525 |         Ok(None) => return,
    |         ^^ not a tuple struct or tuple variant
    |
help: consider importing one of these items instead
    |
6   | use core::result::Result::Ok;
    |
6   | use std::result::Result::Ok;
    |

error[E0532]: expected tuple struct or tuple variant, found function `Ok`
   --> testing-infra/transactional-test-runner/src/framework.rs:526:9
    |
526 |         Ok(Some(s)) => s,
    |         ^^ not a tuple struct or tuple variant
    |
help: consider importing one of these items instead
    |
6   | use core::result::Result::Ok;
    |
6   | use std::result::Result::Ok;
    |

error[E0532]: expected tuple struct or tuple variant, found function `Ok`
  --> testing-infra/transactional-test-runner/src/tasks.rs:35:16
   |
35 |         if let Ok(addr) = parse_address_literal(s) {
   |                ^^ not a tuple struct or tuple variant
   |
help: consider importing one of these items instead
   |
6  | use core::result::Result::Ok;
   |
6  | use std::result::Result::Ok;
```
I have no idea why this compiles inside the Diem repo though...

Reply:
Just a wild guess, but: were you using the same Rust compiler version when compiling outside of the Diem repository? I frequently forget that Diem as a `rust-toolchain` file at its root that pins the toolchain selected to 1.56.1. Maybe your "default" toolchain outside of Diem is one that doesn't compile this code? My default outside of Diem is 1.57.0 -- but that compiles this just fine.

I am curious to understand why compiling this crate outside of Diem would have any impact on this. But, I also personally prefer the explicit `use` syntax over `*`, so I'm OK with this change!

Reply:
> Just a wild guess, but: were you using the same Rust compiler version when compiling outside of the Diem repository? I frequently forget that Diem as a rust-toolchain file at its root that pins the toolchain selected to 1.56.1. Maybe your "default" toolchain outside of Diem is one that doesn't compile this code? My default outside of Diem is 1.57.0 -- but that compiles this just fine.

This is also my original guess. I also used 1.57.0 initially and got the compile errors, so I copied `rust-toolchain` into the new workspace, but the errors persisted. I wondered if something went wrong during this switch, but it didn't seem likely to me since `cargo clean` didn't resolve the issue... Anyway, I'm curious to find out what caused this too.